apps/air-discount-scheme/backend/src/app/modules/user/user.service.ts

```diff
@@ -82,4 +93,17 @@ export class UserService {
   async getUserInfoByNationalId(nationalId: string): Promise<User | null> {
     return this.getUserByNationalId<User>(nationalId, User)
   }
+
+  async getMultipleUsersByNationalIdArray(
+    ids: string[],
+  ): Promise<Array<User> | null> {
```

Comment:
```suggestion
  ): Promise<Array<User | null>> {
```
Never returns null, but can return null inside the array.